content/blog/posts/A-Message-from-the-Director.rst

```diff
@@ -0,0 +1,66 @@
+A Message from the Director
+===========================
+:date: 2017-08-15
+:slug: a-message-from-the-director
+:author: Lance Albertson
+:img: OSLSignPicture.jpg
+:tag: featured-stories
+
+Since its creation in 2002, the Open Source Lab (OSL) has been a beacon of
+innovation, experiential learning, and hosting at Oregon State University and
+beyond. As director, I’ve had the pleasure of watching the lab and its employees
```

Comment:
Usually if you're referring to the position and not the person, you don't. I think technically either one would be correct, so I'll leave it up to you.